pypeit/spectrographs/keck_deimos.py

```diff
@@ -389,6 +429,16 @@ def get_rawimage(self, raw_file, det):
         based on :func:`pypeit.spectrographs.keck_lris.read_lris`, which
         was based on the IDL procedure ``readmhdufits.pro``.
 
+        .. warning::
+
+            ``PypeIt`` currently *cannot* reduce images produced by
```

Comment:
Can you add this warning to the `deimos.rst` file too?
Thanks.

Reply:
Done.